Fitbit.Portable/Models/HeartActivitiesIntraday.cs

```diff
@@ -19,9 +19,25 @@ public class HeartActivitiesIntraday
     public class DatasetInterval
     {
         public DateTime Time { get; set; }
-        public int Value { get; set;}
+        public int Value { get; set; }
     }
 
+    public class IntradayActivitiesHeart
```

Comment:
All classes should be in their own files :)

Reply:
👍